pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonVersion.java

```diff
@@ -21,6 +21,10 @@
 
     private final String version;
 
+    public static String[] whitelistedPythonVersions = { "2.6", "2.7", "3.4", "3.5", "3.6" };
+    public static String defaultPython2 = "2.6";
+    public static String defaultPython3 = "3.5";
```

Comment:
@ethankhall I'm now seeing what you're saying with "I think this belongs somewhere else".  After discussing with @zvezdan some API changes, I started down the path of making the new attributes on `PythonVersion` non-`static` and `private`.  Now they don't at all feel right there because it means you have to create a new `PythonVersion` instance just to get at the default Python 2 and 3 versions, and the whitelist.  I don't think these values should be tied to a specific instance of `PythonVersion`.

I'm also not sure where to put them.